src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || 'jujuda1993',
```

Comment:
Having a hardcoded default password ('jujuda1993') is a security risk. It's important to ensure that sensitive information like passwords is never hardcoded in the source code. Consider removing the default password and ensuring that the password is always provided through environment variables.